src/main/java/Bonnie/Ui.java

```diff
@@ -0,0 +1,50 @@
+package Bonnie;
+
+import java.util.Scanner;
+
+public class Ui {
+    protected static String username;
+
+    public Ui () {
+    }
+
+    /**
+     * Initializes the chat bot and starts the conversation with the user.
+     * Prompts the user for their name and provides a list of available commands.
+     * Continuously prompts the user for input and calls the Parser to process the input.
+     *
+     * @throws DeadlineFormatException     if the format of the deadline task is incorrect
+     * @throws EmptyTodoException          if the body of a todo task is empty
+     * @throws UnknownCommandException     if the input command is unknown
+     */
+    protected static void init() throws DeadlineFormatException, EmptyTodoException, UnknownCommandException {
+        System.out.println("Hello I'm Bonnie, what is your name?");
+
+        Scanner scannerObj = new Scanner(System.in);
+        String my_username = scannerObj.nextLine();
```

Comment:
```suggestion
        String myUsername = scannerObj.nextLine();
```

minor nitpick, according to variable naming convention.